Runtime/Game/Requests/CatalogRequests.cs

```diff
@@ -349,6 +403,8 @@ public class LootLockerListCatalogPricesResponse : LootLockerResponse
         /// </summary>
         public Dictionary<string /*catalog_listing_id*/, LootLockerCurrencyDetails> currency_details { get; set; }
 
+        public Dictionary<string /*catalog_listing_id*/, LootLockerGroupDetails> group_details { get; set; }
```

Comment:
You've missed adding group_details to the AppendCatalogItems method

Reply:
Ty I added it now!